monodrive:
  host: localhost
  port: 2000
  QualityLevel: Low  # Low or Epic
  SynchronousMode: True
  SendNonPlayerAgentsInfo: True
  NumberOfVehicles: 30
  NumberOfPedestrians: 10
  WeatherId: 1
  Framesperepisode: -1 # -1: run episode indefinitely until interrupted
  Episodes: 2
  SimulatorConfig: simulator.json
  VehicleConfig: single_camera.json

